slices/ovn-central.yaml

```diff
@@ -0,0 +1,10 @@
+package: ovn-central
+
+slices:
+  base:
```

Comment:
I would suggest renaming it to `core` to be more in line with other existing slice definitions.

Reply:
Done